Makefile

```diff
@@ -75,14 +82,14 @@ export OPERATOR_NAMESPACE?=openshift-compliance
 # ======================
 SDK_VERSION?=v0.18.2
 ifeq ($(OS_NAME), Linux)
-    OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/$(SDK_VERSION)/operator-sdk-$(SDK_VERSION)-x86_64-linux-gnu
+    OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/$(SDK_VERSION)/operator-sdk-$(SDK_VERSION)-$(ARCH)-linux-gnu
 else ifeq ($(OS_NAME), Darwin)
     OPERATOR_SDK_URL=https://github.com/operator-framework/operator-sdk/releases/download/$(SDK_VERSION)/operator-sdk-$(SDK_VERSION)-x86_64-apple-darwin
 endif
 
-OPM_VERSION=v1.15.2
+OPM_VERSION=v1.20.0
```

Comment:
@mrogers950 do you suppose this change could bite us while we are using the super outdated operator-sdk as we are?

Reply:
I think this will be OK.

Reply:
LGTM, then